clang/test/CIR/CodeGen/pass-object-size.c

```diff
@@ -0,0 +1,17 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -Wno-unused-value -fclangir-enable -emit-cir %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s -check-prefix=CIR
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -Wno-unused-value -fclangir-enable -emit-llvm %s -o %t.ll
+// RUN: FileCheck --input-file=%t.ll %s -check-prefix=LLVM
+void b(void *__attribute__((pass_object_size(0))));
+void e(void *__attribute__((pass_object_size(2))));
+void c() {
+  int a;
+  int d[a];
+    b(d);
+    e(d);
+}
+
+// CIR: %{{[0-9]+}} = cir.objsize(%{{[0-9]+}} : <!void>, max) -> !u64i
```

Comment:
We usually first check the name of the function and then the rest of the content we want to check. Please add that to both CIR and LLVM checks (or check other tests to see how it's done). I'd also prefer if we could check the connection between objsize argument and its definition.